rllib/algorithms/algorithm_config.py

```diff
@@ -1459,6 +1470,10 @@ def training(
             self.max_requests_in_flight_per_sampler_worker = (
                 max_requests_in_flight_per_sampler_worker
             )
+        if _enable_trainer_runner is not NotProvided:
```

Comment:
Should we call this `_enable_rl_trainer_api` analogous to `_enable_rl_module_api`?